model/feature-flags/registry.yaml

```diff
@@ -93,3 +93,40 @@ groups:
         stability: development
         examples: ["Flag `header-color` expected type `string` but found type `number`"]
         brief: A message explaining the nature of an error occurring during flag evaluation.
+      - id: feature_flag.result.value
+        type: string
+        stability: development
+        examples: ["#ff0000", "1", "true"]
+        brief: The evaluated value of the feature flag represented as a JSON string.
+        note: |
+          With some feature flag providers, feature flag results can be quite large or contain private or sensitive details.
+          Because of this, `feature_flag.result.variant` is often the preferred attribute if it is available.
+
+          It may be desirable to redact or otherwise limit the size and scope of `feature_flag.result.value` if possible.
+          Because the evaluated flag value is unstructured and may be any type, it left to the instrumentation author to determine how best to achieve this.
+      - id: feature_flag.result.type
+        type:
+          members:
```

Comment:
I'm currently missing `null` which is a valid JSON type but is _not_ a valid attribute. @trask what is generally the protocol here? Should I put anything technically representable? Or should I limit to what is allowed in attribute values?

Reply:
@trask @lmolkova is there guidance here?

Reply:
The situation I'm thinking of is that an evaluated feature flag `a` returns the value `null` as the result of the evaluation, which when stringified would result in `feature_flag.result.value="null"`.

```
> JSON.stringify(null)
'null'
```

I guess then `feature_flag.result.type` would be `"null"` as well resulting in the attributes:

```json
{
    "feature_flag.result.value": "null",
    "feature_flag.result.type": "null",
}
```

---

I think I answered my own question. `null` may not be a valid attribute but it is a valid feature flag result value, so we need to account for it.